src/event/event_data/delegated/mod.rs

```diff
@@ -14,17 +14,14 @@ pub struct DelegatedInceptionEvent {
     #[serde(flatten)]
     pub inception_data: InceptionEvent,
 
-    #[serde(rename = "da")]
-    pub seal: LocationSeal,
+    #[serde(rename = "di")]
+    pub delegator: IdentifierPrefix,
 }
 
 #[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
 pub struct DelegatedRotationEvent {
     #[serde(flatten)]
```

Comment:
if there is no difference now between Delegation and Rotation events why keep both? can we use RotationEvent struct wherever needed?